test/helpers/test.go

```diff
@@ -610,4 +645,63 @@ func (t *TestOptions) ExpectUrlReturns(url string, expectedStatusCode int, maxDu
 	return RetryExponentialBackoff(maxDuration, f)
 }
 
+func (t *TestOptions) CreateChatOpsCommands(commands []string) error {
+	gitProvider, err := t.GetGitProvider()
+	if err != nil {
+		return err
+	}
+
+	utils.LogInfof("successfully create git provider of kind %s", gitProvider.Kind())
+
+	return nil
+}
+
+// CreateIssueAndAssignToUser creates an issue on the configure git provider and assigns it to a user.
+func (t *TestOptions) CreateIssueAndAssignToUserWithChatOpsCommand(issue *gits.GitIssue, provider gits.GitProvider) error {
+
+	createdIssue, err := provider.CreateIssue(issue.Owner, issue.Repo, issue)
+	if err != nil {
+		return err
+	}
+
+	utils.LogInfof("created issue with number %d\n", *createdIssue.Number)
+
+	err = provider.CreateIssueComment(
+		issue.Owner,
+		issue.Repo,
+		*createdIssue.Number,
+		fmt.Sprintf("/assign %s", provider.CurrentUsername()),
+	)
+	if err != nil {
+		return err
+	}
+	utils.LogInfof("create issue comment on issue %d\n", *createdIssue.Number)
+
+	createdIssue.Owner = issue.Owner
+	createdIssue.Repo = issue.Repo
+
+	return t.ExpectThatIssueIsAssignedToUser(provider, createdIssue, provider.CurrentUsername())
```

Comment:
I feel like this probably needs a fairly short backoff-and-retry - no guarantees how fast Prow will respond.

Reply:
Will do.